src/validators/ArrayValidator.ts

```diff
@@ -13,6 +14,30 @@ export class ArrayValidator<T> extends BaseValidator<T[]> {
 		this.validator = validator;
 	}
 
+	public lengthLt(length: number): this {
+		return this.addConstraint(arrayLengthLt(length) as IConstraint<T[]>);
```

Comment:
@kyranet I'm tasking you to fixing the need for these casts because it's getting annoying to work with

Reply:
Leave it to vladdy to approve a PR and tag Kyra with a request. How long have you know the girl now man. He's never going to do it this way, that said: https://github.com/sapphiredev/shapeshift/issues/43

Reply:
Eh, I can take a look at it too